tests/unit/onyxTest.js

```diff
@@ -24,11 +24,36 @@ Onyx.init({
 describe('Onyx', () => {
     let connectionID;
 
+    /** @type OnyxCache */
+    let cache;
+
+    beforeEach(() => {
+        cache = require('../../lib/OnyxCache').default;
+    });
+
     afterEach(() => {
         Onyx.disconnect(connectionID);
         return Onyx.clear();
     });
 
+    it('should remove key value from OnyxCache/Storage when set is called with null value', () => Onyx.set(ONYX_KEYS.OTHER_TEST, 42)
+        .then(() => Onyx.getAllKeys())
+        .then((keys) => {
+            expect(keys.includes(ONYX_KEYS.OTHER_TEST)).toBe(true);
+            return Onyx.set(ONYX_KEYS.OTHER_TEST, null);
+        })
+        .then(() => {
+            // Checks if cache value is removed.
+            expect(cache.getAllKeys().length).toBe(0);
+
+            // When cache keys length is 0, we fetch the keys from storage.
+            return Onyx.getAllKeys();
+        })
+        .then((keys) => {
+            expect(keys.includes(ONYX_KEYS.OTHER_TEST)).toBe(false);
+            return Onyx.set(ONYX_KEYS.OTHER_TEST, 42);
```

Comment:
Is this necessary? I don't think this value is ever verified anywhere, and the test is over at this point and then Onyx is cleared in `afterEach()`

Reply:
I don't think its absolutely necessary. I just added it to reset the OTHER_TEST state which i removed. 
Anyways added a commit to remove it since removing it doesn't have any effect.